metrix-mapping/src/main/java/com/powsybl/metrix/mapping/log/EmptyFilter.java

```diff
@@ -0,0 +1,25 @@
+package com.powsybl.metrix.mapping.log;
+
+public class EmptyFilter extends AbstractLogBuilder implements LogDescriptionBuilder {
+
+    private String timeSeriesName;
+
+    private double timeSeriesValue;
+
+    public EmptyFilter timeSeriesName(String timeSeriesName) {
+        this.timeSeriesName = timeSeriesName;
+        return this;
+    }
+
+    public EmptyFilter timeSeriesValue(double timeSeriesValue) {
+        this.timeSeriesValue = timeSeriesValue;
+        return this;
+    }
+
+    public EmptyFilter build() {
+        this.message = "Impossible to scale down " + formatDouble(timeSeriesValue) + " of ts '" + timeSeriesName +
```

Comment:
QCA : RMQ => what do you think of String.format( "{a: %s, b: %s, c: %s}", a, b, c ); to avoid this type of concatenation